daemon/container_unix.go

```diff
@@ -896,8 +900,6 @@ func (container *Container) configureNetwork(networkName, service, networkDriver
 		}
 	}
 
-	container.updateSandboxNetworkSettings(sb)
```

Comment:
@estesp updated my patch https://github.com/mavenugo/docker/commit/508c0f979d43ced0c45fd8abfdaff63c7902f3c2 with the above change. PTAL and use it if you agree.